api/router/router.go

```diff
@@ -1,9 +1,9 @@
 package router
 
 import (
-	"log"
-	"tat_gogogo/infrastructure/api/handler"
```

Comment:
and because our architecture is clean architecture so, `Infrastructure layer` is important. Please put api folder back to Infrastructure.

Reply:
Sorry, i don't konw "clean architecture".
I fix this.